app/components/ui/molecules/StaffCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropType from 'prop-types'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+const StaffImg = styled.img`
+  max-width: 100%;
+  outline: 1px solid #e0e0e0;
+  margin-bottom: ${th('space.1')};
+`
+
+const StaffName = styled.div`
+  font-weight: bold;
+`
+const StaffCard = ({ name, telephone, jobTitle, photoURL }) => (
+  <React.Fragment>
```

Comment:
I suppose it depends on how you were thinking of making the StaffGrid component. But if you were imaging something similar to what's in styleguidist, then it might help to line things up by giving this component a specific height (and perhaps min-wdith)? So perhaps it would be worth making this an actual div (or something else) rather than a fragment, so you could apply sizing?

What do you think?